### ISSUE TITLE
[Question] how different the process might be if I replicate this with an Orange pi 0?

### ISSUE THREAD

**Issue by anasouardini** [evanman83/OURS-project#7]
I'm thinking about using this for smaller phone for convenience.

**Comment by evanman83** [comment]
Yes, great idea. We have been discussing which board to use for version 2, to make the phone smaller, faster and more power efficient. We had many discussions on our Telegram group (come join us - see link on main page). We decided on the OrangePi CM4 unit for now, so the problems we are solving regarding the graphics drivers and OS will be relevant to you if you want to use the Zero. We also have a link to a spreadsheet in the Telegram group description which shows all the boards we discussed.

**Comment by NellowTCS** [comment]
I was just about to suggest using a cm4 or cm5! Maybe you could use the Raspberry Pi CM4/5 with Konsta's amazing Android port? (https://konstakang.com/devices/rpi4/ and https://konstakang.com/devices/rpi5/)

**Comment by evanman83** [comment]
The raspberry Pi uses too much power to ever be a portable device with significant battery life, hence we chose the OrangePi CM4 because it has a functional sleep mode.

**Comment by NellowTCS** [comment]
Good point!
